runtime/shared_common/Manager.hpp

```diff
@@ -294,7 +294,9 @@ class SH_Manager
 
 private:
 	UDATA _state;
+#if defined(J9SHR_CACHELET_SUPPORT)
 	bool _allCacheletsStarted;
+#endif
```

Comment:
Is this a general bug that OSX exposed by not having this flag?

Reply:
clang is more picky about this then gcc has been

Reply:
ok